phoenix-core/src/main/java/org/apache/phoenix/coprocessor/MetaDataEndpointImpl.java

```diff
@@ -2340,7 +2365,7 @@ public void createTable(RpcController controller, CreateTableRequest request,
                 // 3. Finally write the mutations to create the table
 
                 // From 4.15 the parent->child links are stored in a separate table SYSTEM.CHILD_LINK
-                // TODO remove this after PHOENIX-4763 is implemented
+                // TODO remove this after PHOENIX-4763 is implemented -> Remove below?
```

Comment:
@twdsilva Is this comment just to keep track of the fact that we need to remove this after all of splittable SYSCAT is implemented or is there anything specific to PHOENIX-4763?